src/autoload/MultiplayerInfo.gd

```diff
@@ -54,3 +55,6 @@ remote func register_player(info):
 	print("Player %d has info %s" % [id, info])
```

Comment:
Are we going to remove the debug prints?

Reply:
Later probably